quote-server/server.js

```diff
@@ -7,17 +7,26 @@ import express from "express";
 import quotes from "./quotes.json" assert { type: "json" };
 
 const app = express();
-// Now register handlers for some routes:
-//   /                  - Return some helpful welcome info (text)
-//   /quotes            - Should return all quotes (json)
-//   /quotes/random     - Should return ONE quote (json)
+
 app.get("/", (request, response) => {
-  response.send("Neill's Quote Server!  Ask me for /quotes/random, or /quotes");
+  response.send("Hadika's Quote Server!  Ask me for /quotes/random, or /quotes");
+});
+app.get("/quotes", (req, res) => {
+  res.send(quotes);
+})
+app.get("/quotes/random", (req, res) => {
+  res.send(pickFromArray(quotes));
+})
+app.get("/quotes/search", (req, res) => {
+  let searchInput = req.query.term.toLowerCase();
```

Comment:
What would happen if a client didn't specify a `term` query at all? e.g. if they just made a request to `http://localhost:3001/quotes/search`?